<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DesktopDialog</class>
 <widget class="QDialog" name="DesktopDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Application Menu Shortcut</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLineEdit" name="inp_name">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>20</y>
        <width>281</width>
        <height>39</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lb_name">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_description">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Description</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="inp_description">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>60</y>
        <width>281</width>
        <height>39</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lb_icon">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Categories (; separated)</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="inp_categories">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>100</y>
        <width>281</width>
        <height>39</height>
       </rect>
      </property>
      <property name="text">
       <string>CasualRDH;Utility;</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_appicon">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>20</y>
        <width>128</width>
        <height>128</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>128</width>
        <height>128</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="inp_comment">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>180</y>
        <width>431</width>
        <height>39</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lb_comment">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Comment</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_command">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Command</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="inp_command">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>220</y>
        <width>431</width>
        <height>39</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lb_categories_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>App icon</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="inp_icon">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>140</y>
        <width>281</width>
        <height>39</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btn_save">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Create Menu Shortcut</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_dismiss">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
